connext/nxtp
# Rinkeby to kovan e2e

## The Problem




- Rinkeby -> kovan `xcall` did not work because the `TestERC20` did not implement `IBridgeToken`

## The Solution




- Update `TestERC20` to implement `IBridgeToken`
- Redeploy `TestERC20` to rinkeby
- Update `TokenRegistry` on rinkeby to point to new token
- Setup new token on `Connext`

**NOTE:** The token was **not** redeployed to kovan, meaning the `TestERC20` on kovan and the `TestERC20` on rinkeby have different interfaces. This is not a problem in the flow as the kovan `TEST` is canonical so does not need to implement the `IBridgeToken` interface, but is something to keep in mind. Chose to not redeploy to kovan to ensure the liquidity is okay

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
